src/screens/ProfileScreen/index.js

```diff
@@ -709,7 +712,12 @@ const ProfileScreen = ({route}) => {
       ) : null}
 
       <BlockComponent ref={refBlockComponent} refresh={getMyFeeds} screen="my_profile" />
-    </View>
+      <PostOptionModal
+        isOpen={isPostOptionModalOpen}
+        onClose={onHeaderOptionClosed}
+        onDeleteClicked={onDeletePost}
+      />
+    </SafeAreaProvider>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new `PostOptionModal` component is added. Ensure that it's correctly integrated and doesn't break the existing UI/UX.
```diff
+ <PostOptionModal
+   isOpen={isPostOptionModalOpen}
+   onClose={onHeaderOptionClosed}
+   onDeleteClicked={onDeletePost}
+ />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->